### PR TITLE
Add crystal linux to setup page

### PR DESCRIPTION
**Pull request by axtloss**

### CODE REVIEW

File: data/distro.yml [hunk 1]
```diff
@@ -738,3 +738,32 @@
         <p>Check that in the Flatpak section the box is checked.</p>
         <p>Note: with this Flathub app search will be integrated in Discover, if you want to limit the app search to Flathub you can mark Flatpak as default by clicking on the star.</p>
     </ol>
+
```
Comment (AsciiWolf) [inline data/distro.yml]: Please, remove the four spaces (" ") on this line and replace them with just an empty line. Thanks!

File: data/distro.yml [hunk 2]
```diff
@@ -738,3 +738,32 @@
         <p>Check that in the Flatpak section the box is checked.</p>
         <p>Note: with this Flathub app search will be integrated in Discover, if you want to limit the app search to Flathub you can mark Flatpak as default by clicking on the star.</p>
     </ol>
+    
+- name: "Crystal Linux"
+  logo: "crystallinux.svg"
+  info: >
+    <ol class="distrotut">
+      <h2>Flatpak is installed by default on Crystal Linux, however if you didn't use jade_gui to install crystal or selected not to install it you can set flatpak up by using following commands<h3>
```
Comment (AsciiWolf) [inline data/distro.yml]:
```suggestion
      <h2>Flatpak is installed by default on Crystal Linux. However, if you didn't use jade_gui to install crystal or selected not to install it, you can set Flatpak up by using the following steps.<h3>
```

File: data/distro.yml [hunk 3]
```diff
@@ -738,3 +738,32 @@
...
+      <li>
+        <h2>Install Flatpak</h2>
+        <p>To install Flatpak in Crystal Linux, you must first make sure your packages are up to date, run the following in a terminal:</p>
```
Comment (AsciiWolf) [inline data/distro.yml:749]:
```suggestion
        <p>To install Flatpak in Crystal Linux, you must first make sure your packages are up to date. Run the following in a terminal:</p>
```

File: data/distro.yml [hunk 4]
```diff
@@ -738,3 +738,32 @@
...
+      <li>
+        <h2>Add the Flathub repository</h2>
+        <p>Flathub is the best place to get Flatpak apps. To enable it, run:</p>
+        <pre><code>
+          <span class="unselectable">$</span> flatpak remote-add --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo
+        </code></pre>
+      </li>
+      <li>
+        <h2>Restart</h2>
+        <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
```
Comment (AsciiWolf) [inline data/distro.yml]: Is graphical installation of Flatpak apps (via GNOME Software, KDE Discover etc.) available in Crystal Linux?

Comment (axtloss) [reply data/distro.yml]: Yes, it installs gnome software by default on gnome

Comment (AsciiWolf) [reply data/distro.yml]: Does the GNOME Software comes with Flatpak plugin preinstalled by default (or as a dependency of the `flatpak` package)?

Comment (axtloss) [reply data/distro.yml]: Yes, when installing gnome software flatpak works directly

Comment (AsciiWolf) [reply data/distro.yml]: Ok, thanks!

**Comment by AsciiWolf** [comment]
By the way, as I mentioned under another PR, I probably won't be merging this PR at the moment. There is already too many distros in the list (see https://github.com/flatpak/flatpak.github.io/issues/471) and I am not sure how popular is Crystal Linux (it looks to be a new distro). Anyway, feel free to keep the PR open, I may merge it in (near) future. Thanks!

File: data/distro.yml [hunk 5]
```diff
@@ -794,3 +794,61 @@
...
+      </li>
+    </ol>
```
Comment (AsciiWolf) [inline data/distro.yml]:
```suggestion
```

Duplicated for no reason.

**Comment by AsciiWolf** [comment]
@axtloss This is probably fine to be merged now after the merge conflict and the [duplicate entry issue](https://github.com/flatpak/flatpak.github.io/pull/547#discussion_r1547834246) are resolved.

File: data/distro.yml [hunk 6]
```diff
@@ -794,3 +794,61 @@
...
+        <p>Then install Flatpak:</p>
+          <pre><code>
+          <span class="unselectable">$</span> ame ins flatpak
+        </code></pre>
+      </li>
+      <li>
+        <h2>Add the Flathub repository</h2>
+        <p>Flathub is the best place to get Flatpak apps. To enable it, run:</p>
+        <pre><code>
+          <span class="unselectable">$</span> flatpak remote-add --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo
```
Comment (AsciiWolf) [inline data/distro.yml:847]:
```suggestion
          <span class="unselectable">$</span> flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
```

Remove the .dl subdomain to be more consistent with other distro entries.

**Comment by axtloss** [comment]
Applied your suggestion and the merge conflict should be resolved too now

File: data/distro.yml [hunk 7]
```diff
@@ -542,3 +542,34 @@
...
+        </code></pre>
+      </li>
+      <li>
+        <h2>Restart</h2>
+        <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
+      </li>
+    </ol>
```
Comment (AsciiWolf) [inline data/distro.yml]:
```suggestion
- name: Crystal Linux
  logo: "crystallinux.svg"
  introduction: >
    <h2>Flatpak is installed by default on Crystal Linux.<h2>
    <p>If you didn't use jade_gui to install crystal or selected not to install it, you can set Flatpak up by using the following steps.<p>
  steps:
    - name: Install Flatpak
      text: "
        <p>To install Flatpak in Crystal Linux, you must first make sure your packages are up to date, run the following in a terminal:</p>
        <terminal-command>ame upg</terminal-command>
        <p>Then install Flatpak:</p>
        <terminal-command>ame ins flatpak</terminal-command>"
    - name: Add the Flathub repository
      text: "
        <p>Flathub is the best place to get Flatpak apps. To enable it, run:</p>
        <terminal-command>flatpak remote-add --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo</terminal-command>"
    - name: Restart
      text: '
        <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>'
```

I forgot that it also needs to be rewritten into a new format.

Comment (AsciiWolf) [reply data/distro.yml]: @axtloss I will merge the PR after these changes are applied.

Comment (AsciiWolf) [reply data/distro.yml:575]: @axtloss Ping. :-)

File: data/distro.yml [hunk 8]
```diff
@@ -542,3 +542,25 @@
...
+
+- name: Crystal Linux
+  logo: "crystallinux.svg"
+  introduction: >
+    <h2>Flatpak is installed by default on Crystal Linux.<h2>
+    <p>If you didn't use jade_gui to install crystal or selected not to install it, you can set Flatpak up by using the following steps.<p>
+  steps:
+    - name: Install Flatpak
+      text: "
+        <p>To install Flatpak in Crystal Linux, you must first make sure your packages are up to date, run the following in a terminal:</p>
```
Comment (AsciiWolf) [inline data/distro.yml]:
```suggestion
        <p>To install Flatpak in Crystal Linux, you must first make sure your packages are up to date. Run the following in a terminal:</p>
```

One more small thing.

Comment (AsciiWolf) [reply data/distro.yml:556]: @axtloss

File: data/distro.yml [hunk 9]
```diff
@@ -542,3 +542,25 @@
       text: '
         <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
         <p>Note: graphical installation of Flatpak apps may not be possible with GNU Guix.</p>'
+      </li>
+    </ol>
```
Comment (AsciiWolf) [inline data/distro.yml]:
```suggestion
```

These two lines also need to be removed.

Comment (AsciiWolf) [reply data/distro.yml:546]: @axtloss One more ping. :-)

**Comment by AsciiWolf** [comment]
Looks good now, thanks! :-)

**Comment by AsciiWolf** [comment]
@axtloss Looks like [it](https://flatpak.org/setup/Crystal%20Linux) sadly is broken. :-(

screenshot

**Comment by AsciiWolf** [comment]
Ah, I see the problem. There is a <h2> tag instead of </h2>. Will fix it.

**Pull request merged**